modules/gltf/register_types.cpp

```diff
@@ -64,22 +64,19 @@ static void _editor_init() {
 
 	// Blend to glTF importer.
 
-	bool blend_enabled = GLOBAL_GET("filesystem/import/blend/enabled");
+	bool blend_enabled = GLOBAL_GET("filesystem/import/blender/enabled");
 	// Defined here because EditorSettings doesn't exist in `register_gltf_types` yet.
-	String blender_path = EDITOR_DEF_RST("filesystem/import/blend/blender_path", "");
+	EDITOR_DEF_RST("filesystem/import/blender/blender3_path", "");
```

Comment:
Blender took a decade to bump from 2 to 3, but what's to say they won't go fast to Blender 4 with minimal compat breakage? :)
I understand you want to make it explicit since many still use 2.79 because 2.8 was too big a jump.